### PR TITLE
CRA: Fix CSS imports in storybook directory

### LINKED ISSUE #69: examples/cra-ts does not work
1. Clone this project.
2. yarn
3. cd examples/cra-ts
4. yarn storybook
5. error:
```
Module parse failed: Unexpected token (7:20)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default { title: 'Button' };
| 
> export const Default: FC = () => (
|   {process.env.REACT_APP_TEST_VAR}
| );
```

# 
I found that `preset-create-react-app` does not support `tsx` ?

```
Module parse failed: Unexpected token (62:4)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| storiesOf('react-photo-view', module)
|   .add('Example', () => (
>     <PhotoProvider>
|       <ImageList>
|         {photoImages.map((item, index) => (
```

### PR DESCRIPTION
**Pull request by mrmckeb**
This PR aims to resolves #69.

It should also resolve:
- Issues discussed here: https://gist.github.com/shilman/bc9cbedb2a7efb5ec6710337cbd20c0c
- https://github.com/storybookjs/storybook/issues/9016

Other work in this PR:
- Dependencies bumped for example projects and preset.
- Unneeded dependencies removed.
- Normalised `cra` and `cra-ts` example projects.

Finally, the CI was updated to run 'build' on all projects to ensure that new PRs don't break builds.

### CODE REVIEW

**Comment by kusmierz** [comment]
awesome, thanks :) is there any plan to release it soon as well?

File: examples/cra/.storybook/config.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import { configure } from '@storybook/react';
+import '../src/App.css';
```
Comment (kusmierz) [inline examples/cra/.storybook/config.js]: btw, is it only me, or is it not possible to include an `.css` file from `examples/cra/.storybook` directory?

**Comment by shilman** [comment]
@kusmierz released as 1.3.2

File: examples/cra/.storybook/config.js (hunk 1 above)
Comment (mrmckeb) [reply examples/cra/.storybook/config.js]: Hi @kusmierz, do you mean you want to include files from the Storybook config folder into your app? That's not possible, but the other way around (as in this example) is.

**Pull request merged**